packages/hdwallet-core/src/ethereum.ts

```diff
@@ -160,6 +161,7 @@ export interface ETHWalletInfo extends HDWalletInfo {
 
 export interface ETHWallet extends ETHWalletInfo, HDWallet {
```

Comment:
Should this be changed to `EvmWallet`?

Reply:
> Should this be changed to `EvmWallet`?

Yes, but it would be a massive breaking change. It would be better to just add a JSDoc comment above explaining that it's really for any Ethereum-compatible chain.